src/terrain/terrain.js

```diff
@@ -397,6 +397,10 @@ export class Terrain extends Elevation {
         return this.enabled ? this.sourceCache : null;
     }
 
+    _isMockSource(): boolean {
```

Comment:
Nit: This function returns true when in globe with no terrain, no? Thoughts on clarifying the function name or adding a brief comment to add context?